[Bug] Cannot add shortcut

### Steps to reproduce/复现步骤

1. Tap add shortcut in LSPosed manager app.

### Expected behaviour/预期行为

Add a shortcut on launcher.

### Actual behaviour/实际行为

Nothing happened.

### Xposed Module List/Xposed 模块列表

```shell
1. I am not disabled.
2. Status bar lyrics.
```


### Magisk Module List/Magisk 模块列表

```shell
1. AirMusic
2. Move Certificates
3. Scene5
4. Shamiko
5. Universal SafetyNet Fix
6. Zygisk - LSPosed
```


### LSPosed version/LSPosed 版本

v1.7.2 (6432)

### Android version/Android 版本

11 (ZUI 13 Stable)

### Magisk version/Magisk 版本

24.1

### Riru version/Riru 版本

N/A

### Version requirement/版本要求

- [x] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

[LSPosed_2022-02-14T17_50_30.414.zip](https://github.com/LSPosed/LSPosed/files/8059733/LSPosed_2022-02-14T17_50_30.414.zip)

There is an exception occurred in logs when creating a shortcut.

By the way, I can add shortcut normally when using previous version (1.7.1 released on github).

```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Configuration android.app.ActivityThread.getConfiguration()' on a null object reference
	at android.content.res.Resources.getDisplayMetrics(Resources.java:2061)
	at android.content.res.TypedArray.<init>(TypedArray.java:1382)
	at android.content.res.TypedArray.obtain(TypedArray.java:53)
	at android.content.res.Resources.obtainAttributes(Resources.java:1991)
	at android.graphics.drawable.Drawable.obtainAttributes(Drawable.java:1621)
	at android.graphics.drawable.Drawable.inflate(Drawable.java:1452)
	at android.graphics.drawable.LayerDrawable.inflate(LayerDrawable.java:180)
	at android.graphics.drawable.DrawableInflater.inflateFromXmlForDensity(DrawableInflater.java:144)
	at android.graphics.drawable.Drawable.createFromXmlInnerForDensity(Drawable.java:1402)
	at android.graphics.drawable.Drawable.createFromXmlForDensity(Drawable.java:1361)
	at android.content.res.ResourcesImpl.loadXmlDrawable(ResourcesImpl.java:974)
	at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:897)
	... 13 more
```

--- a/daemon/src/main/java/org/lsposed/lspd/service/ServiceManager.java
+++ b/daemon/src/main/java/org/lsposed/lspd/service/ServiceManager.java
@@ -19,6 +19,7 @@
 
 package org.lsposed.lspd.service;
 
+import android.app.ActivityThread;
 import android.content.Context;
 import android.ddm.DdmHandleAppName;
 import android.os.IBinder;
@@ -93,6 +94,8 @@ public class ServiceManager {
             Log.e(TAG, "Uncaught exception", e);
             System.exit(1);
         });
+
+        ActivityThread.systemMain();
 
         logcatService = new LogcatService();
         logcatService.start();

